File: week1/pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    //Prompt user for imput

    int Height;

    do
    {
        Height = get_int("Please, enter the number of bricks (between 1 and 8): \n");
    }
    while (Height < 1 || Height > 8);

    //Building the pyramid

    for (int i = 1; i <= Height; i++)
    {
        //Prints left blank space
        for (int j = i; j < Height; j++)
        {
            printf(" ");
        }

        //Prints left blocks
        for (int j = 1; j <= i; j++)
        {
            printf("#");
        }

        //Prints gap
        printf("  ");

        //Prints right blocks
        for (int j = 1; j <= i; j++)
        {
            printf("#");
        }

        //Move prompt to the next line
        printf("\n");
    }
}

File: week6/pset6/mario/less/mario.py
from cs50 import get_int

# Prompt user for imput

Height = 0

while Height < 1 or Height > 8:

    Height = get_int("Please, enter the number of bricks (between 1 and 8):")

# Building the pyramid

for i in range(Height):

    # Prints left blank space

    for j in range(Height - (i + 1)):

        print(" ", end="")

    # Prints left blocks

    for k in range(i + 1):

        print("#", end="")

    print()

